.github/workflows/main.yml

```diff
@@ -0,0 +1,41 @@
+# This workflow will install RST and eventually run tests
+
+name: Compilation Test
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"]
+        cc: ["gcc-default"]
+        cdf: ["39_0"]
+
+    name: ${{ matrix.cc }} on ${{ matrix.os }} with CDF ${{ matrix.cdf }}
+    runs-on: ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Install Ubuntu dependencies
+      if: startsWith(matrix.os, 'ubuntu')
+      run: |
+        sudo apt-get update
+        cat requirements.ubuntu | xargs sudo apt-get install
```

Comment:
Is it possible to rename `requirements.ubuntu` as `.requirements.ubuntu` (ie make it a hidden file), just to help reduce the clutter in the top-level `rst` directory?